docs/msiv1_token_revocation.md

```diff
@@ -47,6 +47,8 @@ Steps 5-9 are new and show how the RP propagates the revocation signal.
 6. **MITS** is basically a proxy, forwarding the query to **SFRP**.
 7. **SFRP** uses MSAL again to get a **new** token from eSTS.
 
+> [!IMPORTANT]
+> This design is only applicable to MIRP api-version=2025-03-30 (for App Service). api-version for service fabric will be soon made available. 
```

Comment:
Chatted with SF folks before. They do not intend to bump their API version, so, it will remain unchanged (as 2019-07-01-preview).

Reply:
Well, I meant to suggest a modification.
```suggestion
> This design is only applicable to MIRP api-version=2025-03-30 (for App Service). api-version for service fabric will remain unchanged as `2019-07-01-preview`. 
```

If you are seeking confirmation from SF, we should have not merged this PR until their confirmation comes back.